src/Classes/Options/Interactivity/Events/DivEvent.ts

```diff
@@ -0,0 +1,31 @@
+import {IDivEvent} from "../../../../Interfaces/Options/Interactivity/Events/IDivEvent";
+import {DivMode} from "../../../../Enums/Modes/DivMode";
+
+export class DivEvent implements IDivEvent {
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new elementId
+     */
+    public get el(): string {
+        return this.elementId;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new elementId
+     * @param value
+     */
+    public set el(value:string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected onespace after colon in parameter](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)